setup.cfg

```diff
@@ -6,19 +6,19 @@ description = Application for upgrading Charmed Openstack
 long_description = file: README.md
 long_description_content_type = text/markdown; charset=UTF-8; variant=GFM
 author = Canonical DevOps Centres
-license = GPLv3
+license = Apache-2.0
```

Comment:
A change in licence seems out of scope for this PR. Also: why?

Reply:
the headers of other files in this PR are also still mentioning the GPL

Reply:
I'll remove that part and open another pr for license.

Reply:
removed